src/frontend/src/lib/utils/token.utils.ts

```diff
@@ -22,7 +24,14 @@ export const getMaxTransactionAmount = ({
 	tokenDecimals: number;
 	tokenId: TokenId;
 }): number => {
-	if (isSupportedEthTokenId(tokenId)) {
+	// TODO: When we will improve the input fee, we should improve the function to receive more token
+	// fee data (such as feeTokenId) and compare with the balance token, instead of having a series
+	// of checks.
+	if (
+		isSupportedEthTokenId(tokenId) ||
```

Comment:
Instead of `tokenId` you can use and pass `tokenStandard`. That way you can do:

```
if (tokenStandard === 'erc20') {
   return Math.max(Number(balance - fee), 0) / 10 ** tokenDecimals;
```

or if you want to inline:

```
return Math.max(Number(balance - (tokenStandard === "erc20" ? fee : 0n)), 0) / 10 ** tokenDecimals
```

Reply:
even better! thanks!